src/Containers/Doctors/Doctors.js

```diff
@@ -36,15 +37,18 @@ const Doctors = () => {
     )
   })
 
+  const { darkMode, light, dark } = useContext(ThemeContext);
+  const theme = darkMode ? dark : light;
+
   return(
-    <>
+    <div className='doctors-container' style={{ color: theme.color, background: theme.background }}>
       <h2 className='dr-sub-title'>Doctors here</h2>
       <section>{allDoctors}
       <Link to='/'>
       <button className='home-button'>Home</button>
       </Link>
       </section>
-    </>
+    </div>
```

Comment:
This looks good for the doctors component. I think we're going to want to do something similar with the MentalHealthPros component for this feature